src/core/discrete_space.jl

```diff
@@ -354,12 +354,21 @@ function get_node_contents(coords::Tuple, model::ABM{A,<:DiscreteSpace}) where {
 end
 
 """
-    get_node_agents(x, model)
+    get_node_agents(x, model) → agents
+
 Same as `get_node_contents(x, model)` but directly returns the list of agents
 instead of just the list of IDs.
+
+    get_node_agents(x::Array{Any, 1}, model) → agents
+
+Call `get_node_contents(x, model)` by broadcasting a list of nodes or agents.
+Useful to obtain a list of agents with their relative direction from starting 
+point `x`.
 """
 get_node_agents(x, model::ABM{A,<:DiscreteSpace}) where {A} = [model[id] for id in get_node_contents(x, model)]
 
+get_node_agents(x::Array{Tuple{Int, Int}, 1}, model::ABM{A,<:DiscreteSpace}) where {A} = get_node_agents.(x, Ref(model))
```

Comment:
`Array{Tuple{Int, Int}, 1}` will only allow 2D grids here right?

Reply:
Yes. Do you think that's too restrictive? My reasoning was that allowing 3D grid positions would result in a more complicated docstring. Maybe we can introduce a second method for `Array{Tuple{Int, Int, Int}, 1}` with its own docstring section so that the specifics of 3D grids can be discussed there?

Reply:
I've never worked with 3D grids on Agents.jl. Can anyone of you fill out the `PLACEHOLDER` that I've inserted in the docstring of the 3D method, please? @Libbum or @Datseris maybe?

Reply:
Why not keeping it general `Array{NTuple{N, Int}, 1} where N`

Reply:
For the method itself it might make sense to keep it as general as possible. For documentation purposes I think it's not so helpful to conflate the methods in one big docstring block. Maybe that's how it is regularly done and if so, I'm of course OK with that as well. However, being more explicit is better here, I think.

Reply:
The constructor that @kavir1698 suggested is the way this should go.

As for doc strings: we want to make them as short and direct as possible. Since we already speak about agent positions a lot in the greater documentation, it's not as important to list the explicit possibilities. As an aside: we can also have 0D and 1D grids, so this further exacerbates the issue.

I would suggest 
```
    get_node_agents(x::Vector{Any}, model) → agents

Obtain a list of agents with their relative direction from starting point `x`.
`x` may be either a list of nodes, or a list of agents.
On a 2D grid for example, agent lists will be returned in a [N, W, E, S] pattern,
and [NW, N, NE, W, E, SW, S, SE] if `moore = true`.
```

Reply:
Maybe this is too offtopic but I wonder how do neighbors look in a 0D or 1D grid?

Reply:
1D at least you would have left and right. For 0D, I can't think of a case. On the other hand, just because we can't does not mean it doesn't exist. We had a discussion about this in #284 concerning a different issue. We want the framework to not get in the way if a user identifies such a need.